pkg/gateway/gatewaylwm2m/gatewaylwm2m.go

```diff
@@ -0,0 +1,241 @@
+package gatewaylwm2m
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/edgenesis/shifu/pkg/deviceshifu/deviceshifubase"
+	"github.com/edgenesis/shifu/pkg/gateway/gatewaylwm2m/lwm2m"
+	"github.com/edgenesis/shifu/pkg/k8s/api/v1alpha1"
+	"github.com/edgenesis/shifu/pkg/logger"
+	"gopkg.in/yaml.v2"
+	"k8s.io/client-go/rest"
+	"knative.dev/pkg/configmap"
+)
+
+const (
+	deviceShifuHost          = "http://localhost:8080"
```

Comment:
fixed endpoint?

Reply:
yes for deviceshifu HTTP listening port always 8080